src/scripts/main.js

```diff
@@ -1,3 +1,391 @@
+
+  if (findInput) {
+    findInput.remove();
+    e.target.textContent = oldValue;
+  }
+
+  const input = document.createElement('input');
+
+  input.setAttribute('class', 'cell-input');
+  e.target.textContent = '';
+  e.target.append(input);
+  input.focus();
+
+  if (
+    e.target === e.target.parentElement.children[3] ||
+    e.target === e.target.parentElement.children[4]
+  ) {
+    input.style.width = '60px';
+  } else {
+    input.style.width = '100px';
+  }
+
+  input.onkeydown = (even) => {
+    if (even.key === 'Enter') {
+      if (e.target === e.target.parentElement.children[3]) {
+        input.setAttribute('type', 'number');
+
+        if (input.value < 18 || input.value > 90) {
+          e.target.textContent = oldValue;
+          input.remove();
+
+          return;
+        }
+      }
+
+      if (e.target === e.target.parentElement.children[4]) {
+        input.setAttribute('type', 'number');
+
+        if (input.value <= 0) {
+          e.target.textContent = oldValue;
+          input.remove();
+
+          return;
+        } else {
+          e.target.textContent = `$${(+input.value).toLocaleString('en-US')}`;
+          input.remove();
+
+          return;
+        }
+      }
+
+      if (!input.value.trim()) {
+        e.target.textContent = oldValue;
+      } else {
+        e.target.textContent = input.value;
+      }
+
+      input.remove();
+    }
+  };
+
+  input.onblur = () => {
+    if (e.target === e.target.parentElement.children[3]) {
+      input.setAttribute('type', 'number');
+
+      if (input.value < 18 || input.value > 90) {
+        e.target.textContent = oldValue;
+        input.remove();
+
+        return;
+      }
+    }
+
+    if (e.target === e.target.parentElement.children[4]) {
+      input.setAttribute('type', 'number');
+
+      if (input.value <= 0) {
+        e.target.textContent = oldValue;
+        input.remove();
+
+        return;
+      } else {
+        e.target.textContent = `$${(+input.value).toLocaleString('en-US')}`;
+        input.remove();
+
+        return;
+      }
+    }
+
+    if (!input.value.trim()) {
+      e.target.textContent = oldValue;
+    } else {
+      e.target.textContent = input.value;
+    }
+
+    input.remove();
+  };
+});
+
+const createMessagSaccess = (div) => {
+  const titleSuccess = document.createElement('h2');
+
+  titleSuccess.style.fontWeight = '900';
+  titleSuccess.style.fontSize = '20px';
+  titleSuccess.textContent = 'Added successfully';
+  div.style.background = 'rgba(10, 189, 0, 0.3)';
+  div.classList.add('success');
+  div.append(titleSuccess);
+  document.body.append(div);
+};
+
+const createMessageError = (div) => {
+  const titleElement = document.createElement('h2');
+  const titleElementName = document.createElement('h2');
+  const titleElementPosition = document.createElement('h2');
+
+  titleElement.style.fontWeight = '900';
+  titleElement.style.fontSize = '20px';
+
+  if (inputName.value.length < 4) {
```

Comment:
The check for the name length should be consistent with the validation in 'checkValidValueError'. If the minimum length is 4, it should be checked the same way in both places.